### DISCUSSION TITLE
Requesting someoen to sponsor my project

### DISCUSSION THREAD

**Discussion by dhruvabhat24** [community/community#146129] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi 

I hope this message finds you well. My name is Dhruva, and I am the creator of Covert Drug Communication detection a project that aims to detect drug traffic communication in real time in telegram platform.

## Due to security resons the project repo is made private if anyone is interested please contact me. 

If anyone is interested please provide a sponsor to me . 

Thank you for considering this request, and I greatly appreciate your support.

Best regards,  
Dhruva 
[Github](https://github.com/dhruva24)

**Comment by mecodeatlas** [comment]
Hi @dhruvabhat24, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
